dkg/protocol.go

```diff
@@ -10,6 +10,7 @@ type KeyGenOutput struct {
 	Share           *bls.SecretKey
 	OperatorPubKeys map[types.OperatorID]*bls.PublicKey
 	ValidatorPK     types.ValidatorPK
+	Threshold       uint64
```

Comment:
It's included here so that this struct contains sufficient information for the share to operate. It'll be more tedious to get the info if it's scattered across two types. However, if we enforce that threshold = 2f+1, we can derive it from the number of operators which is 3f+1. But including it here gives us more flexibility to allow other thresholds (e.g. 10 out of 13).